src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -179,7 +179,7 @@ public Money EstimateFee(TransactionBuildContext context)
 
         /// <inheritdoc />
         [NoTrace]
-        public void CacheSecret(WalletAccountReference walletAccount, string walletPassword, TimeSpan duration)
+        public SecureString CacheSecret(WalletAccountReference walletAccount, string walletPassword, TimeSpan duration)
```

Comment:
It looks as though no code currently consumes the return value from this method, so I'll write a quick unit test that tests this just to make sure it is good to go.